### DISCUSSION TITLE
Default JSON for dag_run.conf in the UI

### DISCUSSION THREAD

**Discussion by JavierLopezT** [apache/airflow#25365] [General]
### Description

It would be nice to be able to define a default JSON that already appears written in the UI. So for instance, you would define in the code something like:

```
with DAG(blabla) as dag:
    operators
    dag.default_run_conf = '{"date": "2020/01/01", "module": "ff", "serial_number", "123"}'
```

And in the UI, once you click on Trigger DAG with configuration, this would be there automatically:
![image](https://user-images.githubusercontent.com/11339132/181455142-9ec10c9e-60d9-4d61-aedf-63b29d43c29a.png)

   

### Use case/motivation

Saving time. Also, for non-techincal people that trigger DAGs, is more friendly just to fill values than to write a JSON from scratch

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by eladkal** [comment]
Isn't this what Params do?
https://airflow.apache.org/docs/apache-airflow/stable/concepts/params.html

**Comment by uranusjr** [comment]
Yes I think this exactly what Params are supposed to take over for. Maybe we should deprecate `dag_run.conf` altogether and start pointing people to it, or are there still reasons to prefer `conf`?

**Reply by potiuk** [reply]
The only reson why we would like to have params  if they do not work with triggering https://airflow.apache.org/docs/apache-airflow/stable/dag-run.html#passing-parameters-when-triggering-dags

I believe (but maybe I am wrong) that we can trigger/backfill tasks only via run_conf override: 

 https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#dag-run-conf-overrides-params

So we can't deprecate/remove dag_run.conf until we add a mechanism to set params directly I believe (unless I am wrong about it)

**Reply by uranusjr** [reply]
> set params directly

You mean set params when a run is triggered? Currently you can define a param’s default value in the DAG.
